### DISCUSSION TITLE
Welcome to org-listcruncher Discussions!

### DISCUSSION THREAD

**Discussion by dfeich** [dfeich/org-listcruncher#7] [General]
## 👋 Welcome!
Org-listcruncher was written to provide an easier planning workflow within lists, but I just assembled it as a prototype. Even though I've been using it constantly over the last years and it's mostly ok for my own purposes, I would be interested to hear whether you can think of other use cases and better integrations.

**Comment by linuxclef** [comment]
Thanks for an awesome package @dfeich !

# Integrate with orgmode's column view

One idea I had was to be able to view the output of `org-listcruncher-to-table` as an orgmode overlay rather than just in a dynamic block, maybe by hooking into orgmode's [column view](https://orgmode.org/manual/Column-View.html) so that rather than just showing a single `Tags` column, the column view would also show a column for each org-listcruncher key. That would probably on make sense if https://github.com/dfeich/org-listcruncher/issues/10 could be supported though.

**Reply by dfeich** [reply]
One probably could write a separate overlay based mode yes. What I recently used in a planning session together with a whole team during a video conference was emacs' **hi-lock-mode**. This mode allows you to define a number of patterns which then highlight matches inside the buffer. And, best of all, you can define the patterns in comment lines inside of your buffer. Here an example of patterns for highlighting entries
```
# Hi-lock: (("\\[.*\\] *$" (0 'hi-blue)))
# Hi-lock: (("\\(-\\|+\\|*\\) t:[^.]*.?" (0 'hi-yellow prepend)))
```
And this will highlight the matching lines and variable sections of the following example list. Just place the above lines into the same buffer and then invoke `M-x hi-lock-mode`.

```
    #+NAME: lst_MinViableSystemDraft
    - [Effort: ?, Who: ?, MatCosts: ?]
      + initial network integration [WP: Netw]
        - t: compile netw requirements based on services to reach.
          [Who: team-A team-B networkers ]
        - t: Decide on initial network integration. E.g just FW rules, routed
          VLAN, ...? [Who: networkers sec-A sec-B]
          - does all traffic have to pass through a FW, do we trust a
            dedicated line? (Costs?). [MatCosts: potentially]
        - t: Security vetting of setup. [Who: sec-A sec-B]
      + provide initial cluster resources. [WP: Vcluster, Who: team-B]
        - t: provide initial virtual cluster basic resources. [MatCosts: yes]
          - compute nodes, login node, service node for batch system
        - t: initial implementation of user/project storage.  [MatCosts: yes]
          - need not be at scale and performance, but POSIX semantics
        - t: provide basic OS provisioning process triggerable by organisation-A.
        - t: provide info/docs for how to access, configure, reinstall, etc.
```

And this would be a code block for invoking org-listcruncher for this kind of list format

```
    #+BEGIN_SRC elisp :results output raw :var listname="lst_MinViableSystem" title="Minimal Viable System" :exports results
      (princ (org-listcruncher-to-table listname
                                        :parsefn (org-listcruncher-mk-parseitem-default
                                                  :tag "t:" :bra "[" :ket "]")
                                        :order '("WP" "description" "Who" "Effort")
                                        :formula "@>$1=Total::@>$4=vsum(@I..@II);N"))
    #+END_SRC
```


Cheers,
Derek
